src/authorize.js

```diff
@@ -37,16 +37,40 @@
       url += '&state=' + encodeURIComponent(redirectUri.substring(hashPos+1));
     }
     url += '&response_type=token';
+
+    if (global.cordova) {
+      return RemoteStorage.Authorize.openWindow(
+          url,
+          redirectUri,
+          'location=yes,clearsessioncache=yes,clearcache=yes'
+        )
+        .then(function (authResult) {
+          remoteStorage.remote.configure({
+            token: authResult.access_token
+          });
+
+          // sync doesnt start until after reload
+          // possibly missing some initialization step?
+          global.location.reload();
+        })
+        .then(null, function (error) {
+          console.error(error);
```

Comment:
Need to change lib state here in addition to (or instead of?) logging to fix the OAuth abortion issue. Not sure how that should be done though.